addons/popochiu/engine/objects/prop/popochiu_prop.gd

```diff
@@ -137,6 +137,98 @@ func get_total_frames() -> int:
 	return frames * v_frames
 
 
+#endregion
+
+#region AnimationPlayer ############################################################################
+## Will play the Popochiu Prop Animation Player [param name] animation on the Popochiu 
+## Prop if the [prop name] animation exists.
+## Optionally you can use other AnimationPlayer options, see Godot AnimationPlayer documentation for
+## more details.
+## [i]This method is intended to be used inside a [method Popochiu.queue] of instructions.[/i]
+func queue_play_animation(name: StringName = &"", custom_blend: float = -1, custom_speed: float = 1.0, from_end: bool = false) -> Callable:
+	return func (): await play_animation(name, custom_blend, custom_speed, from_end)
+
+
+## Will play the Popochiu Prop Animation Player [param name] animation on the Popochiu 
+## Prop if the [prop name] animation exists.
+## Optionally you can use other AnimationPlayer options, see Godot AnimationPlayer documentation for
+## more details.
+func play_animation(name: StringName = &"", custom_blend: float = -1, custom_speed: float = 1.0, from_end: bool = false) -> void:
+	if not has_node("AnimationPlayer"): return
+	$AnimationPlayer.play(name, custom_blend, custom_speed, from_end)
+
+
+## Will Play the Popochiu Prop Animation Player [param name] animation backwards on the Popochiu
+## Prop if the [prop name] animation exits.
+## This method is a shorthand for play_animation() with custom_speed = -1.0 and from_end = true
+## [i]This method is intended to be used inside a [method Popochiu.queue] of instructions.[/i]
+func queue_play_animation_backwards(name: StringName = &"", custom_blend: float = -1) -> Callable:
+	return func (): await play_animation_backwards(name, custom_blend)
+
+
+## Will Play the Popochiu Prop Animation Player [param name] animation backwards on the Popochiu
+## Prop if the [prop name] animation exits.
+## This method is a shorthand for play_animation() with custom_speed = -1.0 and from_end = true
+func play_animation_backwards(name: StringName = &"", custom_blend: float = -1) -> void:
+	if not has_node("AnimationPlayer"): return
+	$AnimationPlayer.play_backwards(name, custom_blend)
+
+
+## Will stop the animation that is currently playing on the Popochiu Prop.
+## The animation position is reset to 0 and the custom_speed is reset to 1.0.
+## Set [param keep_state] to true to avoid the animation to be updated visually.
+## [i]This method is intended to be used inside a [method Popochiu.queue] of instructions.[/i]
+func queue_stop_animation(keep_state: bool = false) -> Callable:
+	return func (): await stop_animation(keep_state)
+
+
+## Will stop the animation that is currently playing on the Popochiu Prop.
+## The animation position is reset to 0 and the custom_speed is reset to 1.0.
+## Set [param keep_state] to true to avoid the animation to be updated visually.
```

Comment:
```suggestion
## Will stop the animation that is currently playing.
## The animation position is reset to [code]0[/code] and the [code]custom_speed[/code] is reset to
## [code]1.0[/code]. Set [param keep_state] to [code]true[/code] to avoid the animation to be
## updated visually.
```